modules/gapi/include/opencv2/gapi/stereo.hpp

```diff
@@ -47,13 +65,12 @@ G_TYPED_KERNEL(GStereo, <GMat(GMat, GMat, const StereoOutputFormat)>, "org.openc
 
 } // namespace calib3d
 
-/** @brief Extract disparity/depth information depending on passed StereoOutputFormat argument.
-The function extracts disparity/depth information depending on passed StereoOutputFormat argument from
-given stereo-pair.
+/** @brief Computes disparity/depth map for the specified stereo-pair.
+The function compute disparity or depth map depending on passed StereoOutputFormat argument.
 
-@param left left 8-bit unsigned 1-channel image of @ref CV_8UC1 type
```

Comment:
Can image now have another type (not CV_8UC1)?

Reply:
No, input image should be a CV_8UC1

Reply:
I think needs leave comment about type.

Reply:
Done